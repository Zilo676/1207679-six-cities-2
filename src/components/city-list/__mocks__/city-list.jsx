import React from "react";

const CityList = () => {
  return <></>;
};

export {CityList};
export default CityList;
