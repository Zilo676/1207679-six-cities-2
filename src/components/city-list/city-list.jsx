import React from 'react';
import PropTypes from 'prop-types';

import {City as city} from '../../mocks/cities.js';
import City from '../city/city.jsx';

const cities = Object.values(city);

const CityList = () => {
  return (<ul className="locations__list tabs__list">{
    cities.map((city, i) => (<City city={city} key={i} />))}
  </ul>)
};

export {CityList};
