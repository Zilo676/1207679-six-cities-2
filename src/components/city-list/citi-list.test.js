import React from 'react';
import renderer from 'react-test-renderer';

import {CityList} from './city-list.jsx';

jest.mock(`../city/city.jsx`, () => `city`);

it(`Correctly rendered`, () => {

  const tree = renderer.create(<CityList cities={[]}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
