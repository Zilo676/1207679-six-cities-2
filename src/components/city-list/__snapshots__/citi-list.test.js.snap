// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly rendered 1`] = `
<ul
  className="locations__list tabs__list"
>
  <city
    city="Paris"
  />
  <city
    city="Colonge"
  />
  <city
    city="Brussels"
  />
  <city
    city="Amsterdam"
  />
  <city
    city="Hamburg"
  />
  <city
    city="Dusseldorf"
  />
</ul>
`;
